app/src/main/java/com/noalecohen/dispatcher/AccountFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.noalecohen.dispatcher
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.noalecohen.dispatcher.databinding.FragmentAccountBinding
+import model.BaseFragment
+
+class AccountFragment : BaseFragment() {
+    private lateinit var binding: FragmentAccountBinding
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        binding = FragmentAccountBinding.inflate(inflater, container, false)
+        displayTitles()
+        return binding.root
+    }
+
+    private fun displayTitles() {
+        var titles = ""
+        for(article in articleList) {
+            if(article.Title != null) {
+                titles += article.Title
+            }
+            titles += "\n\n"
```

Comment:
Is this the best logic to create this string?

Reply:
I guess the answer is 'no'... Can you help me please?